Content.Shared/Throwing/ThrowAttemptEvent.cs

```diff
@@ -11,4 +11,9 @@ public ThrowAttemptEvent(IEntity entity)
 
         public IEntity Entity { get; }
     }
+
+    /// <summary>
+    /// Raised when we try to pushback an entity from throwing
+    /// </summary>
+    public sealed class ThrowPushbackEvent : CancellableEntityEventArgs {}
```

Comment:
Given this is a cancellable event, I would call it ThrowPushbackAttemptEvent. ThrowPushbackEvent implies it has already happened, when it might get cancelled and not occur at all